src/Umbraco.Infrastructure/Runtime/RuntimeState.cs

```diff
@@ -203,11 +201,14 @@ private UmbracoDatabaseState GetUmbracoDatabaseState(IUmbracoDatabaseFactory dat
                         return UmbracoDatabaseState.NeedsUpgrade;
                     }
 
+                    // TODO: We will need to scan for implicit migrations.
 
                     // TODO: Can we save the result of this since we'll need to re-use it?
```

Comment:
I think that is `PackageMigrationState`'s responsibility and it is in the container as a singleton, so if it just saves the result in a field and uses that if it is not null, I guess that should be fine.

Reply:
oops, i left these TODOs in here. it's all taken care of already so i'll just remove the todos